### ISSUE TITLE
[2.x] Regression: PolymerElement incorrectly processed by modulizer after 2.6.0

### ISSUE THREAD

**Issue by web-padawan** [Polymer/polymer#5447]
### Description
The regression breaking `polymer-modulize` was introduced in 748db7c by @aomarks 
Here is the relevant [diff](https://github.com/Polymer/polymer/commit/748db7ce2e0bc57100026083fc2c2cbee58b8603#diff-b3d0398e3be60fcdd00a4938032522ebL33) (especially, the assignment and JSDoc seem related).

After this commit, Polymer 2.6.1 and later is no longer properly converted by modulizer.
The related code in modulizer in [rewrite-namespace-exports.ts](https://github.com/Polymer/tools/blob/ce5ca6cf050e2f72b41cd053c2e59f66bdb0c747/packages/modulizer/src/passes/rewrite-namespace-exports.ts#L249)

### Expected
Confirmed to work at least at the commit 5f5d2c2:
```js
import { PolymerElement } from '@polymer/polymer/polymer-element.js';
```
### Actual
Broken on 2.x released version 2.6.x and 2.7:
```js
import { Element } from '@polymer/polymer/polymer-element.js';
```

This has been reported in Slack many times, and a lot of people suffered from that.
In Vaadin we workaround it by pinning Polymer in out [internal tool](https://github.com/vaadin/magi-cli/blob/f8859580fcc854e7111acf3b382720f8d75a789b/bin/magi-p3-convert#L76) before running modulizer.

/cc @bicknellr who has been working on modulizer at least before Google IO.

**Comment by aomarks** [comment]
Just to understand the impact of this, does this prevent you from using modulizer on your own elements? I naively wouldn't think it matters if Polymer 2 no longer gets modulized correctly, since we've already forked Polymer 3 from modulized Polymer 2. Is the issue that modulizer no longer is able to modulize your elements because it doesn't understand this aspect of Polymer?

**Comment by web-padawan** [comment]
Yes, the problem is that modulizer produces    the `Element` instead of `PolymerElement` when running on our elements like `vaadin-button`.

For now we still keep the Polymer 2 codebase, and run modulizer in CI for tests, and when making new release.

**Comment by aomarks** [comment]
BTW this fix was released with 0.4.2
